package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "patios",
+  "version": "0.0.1",
+  "description": "Manage large Open API specifications and automatically generate TypeScript definitions from it.",
```

This is my fault, I switched from using singular to plural.
```suggestion
  "description": "Manage large Open API specifications and automatically generate TypeScript definitions from them.",
```